README.md

```diff
@@ -34,23 +34,23 @@ A list of comma-separated build arguments.
 
 ### `target`
 
-Specific target stage to build image.
+Specific target stage to build image. Only available with a Dockerfile file.
 
 ### `no-cache`
 
 Set to "true" when no cache should be used when building the image
 
 ### `cache-from`
 
-A list of comma-separated images where cache should be imported from.
+A list of comma-separated images where cache should be imported from. Only available with a Dockerfile file.
```

Comment:
```suggestion
A list of comma-separated images where cache should be imported from. Specific target stage to build image. Only available when `file` refers to a Dockerfile instead of an Okteto Manifest.
```